src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510057>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510058>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510060>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510061>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510062>))
```

Comment:
>A coloring agent which is used to deposit color onto a substrate via an inkjet printer.

Again, the definition pattern is off. Further, this definition doesn't make sense. The colouring agent isn't used to deposit anything. 

Given that you have wax, resin, and aqueous inks noted, it would make sense to include defined (with equivalence axioms) subclasses of ink to contain them. Not essential, but would be good to take the opportunity.